include/lexy/dsl/delimited.hpp

```diff
@@ -57,6 +57,26 @@ struct _del_chars
         {
             reader.bump();
         }
+        // Then, try to match any code point in default_encoding or byte_encoding
+        else if constexpr (std::is_same_v<typename Reader::encoding, lexy::default_encoding> ||
```

Comment:
Please move this inside the `if constexpr` check below that checks the return type - if it's false_type, we don't need to execute this block of code either.

Reply:
Done.